### PR TITLE
Remove dirlight position normalization in examples

### PR DESCRIPTION
**Pull request by mbredif**
As proposed by @WestLangley in #14658, this PR removes the position normalization of DirectionalLights in the examples.

### CODE REVIEW

File: examples/webgldeferred_animation.html [hunk 1]
```diff
@@ -149,7 +149,7 @@
 				for ( var i = 0; i < numLights; i ++ ) {
 
 					var light = new THREE.PointLight( 0xffffff, 2.0, distance );
-					c.set( Math.random(), Math.random(), Math.random() ).normalize();
```
Comment (mrdoob) [inline examples/webgldeferred_animation.html]: This one is normalizing a color 😇

Comment (mbredif) [reply examples/webgldeferred_animation.html]: Good catch!

**Comment by mbredif** [comment]
updated

**Comment by WestLangley** [comment]
You can only remove `normalize()` if `light.target.position` is the origin, and the light is a child of the scene. Otherwise, you are changing the direction of the light.

**Comment by mbredif** [comment]
Thanks for the clarification. That would require a closer inspection indeed...
If any one wants to take over this effort, be welcome !

**Comment by Mugen87** [comment]
Closing. The change needs a rework with @WestLangley's feedback in mind.

**Pull request closed without merging**